src/Microsoft.OData.Cli/ODataCliPackageInstaller.cs

```diff
@@ -0,0 +1,75 @@
+﻿//-----------------------------------------------------------------------------
+// <copyright file="ODataCliPackageInstaller.cs" company=".NET Foundation">
+//      Copyright (c) .NET Foundation and Contributors. All rights reserved. 
+//      See License.txt in the project root for license information.
+// </copyright>
+//----------------------------------------------------------------------------
+
+using System;
+using System.CommandLine;
+using System.Diagnostics;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.OData.CodeGen.PackageInstallation;
+
+namespace Microsoft.OData.Cli
+{
+    /// <summary>
+    /// An implementation of the <see cref="IPackageInstaller"./>
+    /// </summary>
+    public class ODataCliPackageInstaller : IPackageInstaller
+    {
+        private string projectPath;
+        private IConsole console;
+
+        /// <summary>
+        /// Creates an instance of <see cref="ODataCliPackageInstaller"/> 
+        /// </summary>
+        /// <param name="projectPath">The project path</param>
+        /// <param name="console">The <see cref="IConsole"/> object to use for logging.</param>
+        public ODataCliPackageInstaller(string projectPath, IConsole console)
+        {
+            this.projectPath = projectPath;
+            this.console = console;
+        }
+
+        /// <summary>
+        /// Writes a message to the log
+        /// </summary>
+        /// <param name="logMessageCategory">The severity level of the message to be logged</param>
+        /// <param name="message">The message to log</param>
+        /// <param name="args">It contains any other object to format</param>
+        /// <returns></returns>
+        public Task CheckAndInstallNuGetPackageAsync(string packageSource, string packageName)
+        {
+            if (Directory.GetFiles(projectPath, "*.csproj").Length == 0)
+            {
+                return Task.FromResult("No project files. Nuget packages not installed");
+            }
+            else
+            {
+                try
+                {
+                    var process = new Process();
```

Comment:
```suggestion
                    Process process = new();
```
This syntax avoids `var` and it's just as concise :)